cypress/e2e/settings-component.cy.js

```diff
@@ -35,31 +35,31 @@ describe("Tests for settings component", () => {
   });
 
   xit("should allow multiple files to be opened in debug tree when setting is enabled and close all but one report when setting is disabled", () => {
-    cy.get("[data-cy-open-settings-modal]").as("openSettingsModal").click();
-    cy.get("[data-cy-toggle-show-amount]").should(
+    cy.get("[data-cy-debug-table='openSettings']").as("openSettingsModal").click();
+    cy.get("[data-cy-settings-modal='showAmount']").should(
       "have.attr",
       "value",
       "false"
     );
-    cy.get("[data-cy-close-settings-modal]").click();
+    cy.get("[data-cy-settings-modal='close']").click();
     cy.get('[data-cy-record-table-index="0"]').click();
     cy.get('[data-cy-record-table-index="1"]').click();
     cy.get("#debug-tree .jqx-tree-dropdown-root > li").should(
       "have.length",
       "1"
     );
     cy.get("@openSettingsModal").click();
-    cy.get("[data-cy-toggle-show-amount]").click();
-    cy.get("[data-cy-toggle-show-amount]").should("have.attr", "value", "true");
-    cy.get("[data-cy-close-settings-modal]").click();
+    cy.get("[data-cy-settings-modal='showAmount']").click();
+    cy.get("[data-cy-settings-modal='showAmount']").should("have.attr", "value", "true");
+    cy.get("[data-cy-settings-modal='close']").click();
     cy.get('[data-cy-record-table-index="0"]').click();
     cy.get("#debug-tree .jqx-tree-dropdown-root > li").should(
       "have.length",
       "2"
     );
-    cy.get("[data-cy-open-settings-modal]").as("openSettingsModal").click();
-    cy.get("[data-cy-toggle-show-amount]").click();
-    cy.get("[data-cy-close-settings-modal]").click();
+    cy.get("[data-cy-debug-table='openSettings']").as("openSettingsModal").click();
+    cy.get("[data-cy-settings-modal='showAmount']").click();
+    cy.get("[data-cy-settings-modal='close']").click();
```

Comment:
Modal mag je hier weglaten
```suggestion
    cy.get("[data-cy-settings='showAmount']").click();
    cy.get("[data-cy-settings='close']").click();
```